tests/APIv2/MediaTest.php

```diff
@@ -375,6 +375,7 @@ public function provideTypeUrls() {
             ['https://www.instagram.com/p/BizC-PPFK1m', 'instagram'],
             ['https://soundcloud.com/syrebralvibes/the-eden-project-circles', 'soundcloud'],
             ['https://imgur.com/gallery/10HROiq', 'imgur'],
+            ['https://giphy.com/gifs/super-smash-bros-ultimate-jwSlQZnsymUW49NC3R', 'giphy'],
```

Comment:
It looks like your regex in `matchUrl` supports some URL variations. We should test for those additional formats here, so we know if they fail to match in the future.